content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -272,16 +275,83 @@ transaction functions. We recommend you get in and out as quick as possible!
 - [Default transaction isolation level in MySQL](https://dev.mysql.com/doc/refman/8.0/en/innodb-transaction-isolation-levels.html)
+  <br />
+  <br />
+  CockroachDB and PlanetScale always use the <inlinecode>
+    Serializable
+  </inlinecode> isolation level.
+
+</Admonition>
+
+To set the transaction isolation level, use the `isolationLevel` option in the second parameter of the API. For example:
+
+```jsx
+await prisma.$transaction(
+  async (prisma) => {
+    // Code running in a transaction...
+  },
+  {
+    isolationLevel: Prisma.TransactionIsolationLevel.Serializable, // optional
+    maxWait: 5000, // default: 2000
+    timeout: 10000, // default: 5000
+  }
+)
+```
+
+Prisma Client supports the following isolation levels if they are available in the underlying database:
+
+- `ReadCommitted`
+- `ReadUncommitted`
+- `RepeatableRead`
+- `Serializable`
+- `Snapshot`
+
+##### Isolation levels by database
+
+The isolation levels available for each database connector are as follows:
+
+| Database    | `ReadCommitted` | `ReadUncommitted` | `RepeatableRead` | `Serializable` | `Snapshot` |
+| ----------- | --------------- | ----------------- | ---------------- | -------------- | ---------- |
+| PostgreSQL  | ✔️              | ✔️                | ✔️               | ✔️             | No         |
+| MySQL       | ✔️              | ✔️                | ✔️               | ✔️             | No         |
+| SQL Server  | ✔️              | ✔️                | ✔️               | ✔️             | ✔️         |
+| CockroachDB | No              | No                | No               | ✔️             | No         |
+| SQLite      | No              | No                | No               | ✔️             | No         |
+
+By default Prisma Client will set the isolation level to the value currently configured in your database.
+
+The isolation levels configured by default in each database are:
+
+| Database    | Default          |
+| ----------- | ---------------- |
+| PostgreSQL  | `ReadCommitted`  |
+| MySQL       | `RepeatableRead` |
+| SQL Server  | `ReadCommitted`  |
+| CockroachDB | `Serializable`   |
+| SQLite      | `Serializable`   |
+
+For more database-specific information on isolation levels, see the following resources:
 
-## Join the Conversation on GitHub
+- [Transaction isolation levels in PostgreSQL](https://www.postgresql.org/docs/9.3/runtime-config-client.html#GUC-DEFAULT-TRANSACTION-ISOLATION)
+- [Transaction isolation levels in Microsoft SQL Server](https://docs.microsoft.com/en-us/sql/t-sql/statements/set-transaction-isolation-level-transact-sql?view=sql-server-ver15)
+- [Transaction isolation levels in MySQL](https://dev.mysql.com/doc/refman/8.0/en/innodb-transaction-isolation-levels.html)
 
-If you'd like to see transactions supported in the future, [please join the discussion on GitHub](https://github.com/prisma/prisma-client-js/issues/349).
+CockroachDB and PlanetScale always use the `Serializable` isolation level.
```

Comment:
This whole block "For more ..." until "... isolation level" is also still present in the "Before version 4.2.0" admonition above. That repetition is a bit weird and probably unintentional?

Reply:
It's currently intentional as pre-4.2.0 users need to know as well as 4.2.0+ users.

Agree it's a little weird, happy for suggestions on how to reorganise.